src/ontology/eupath_dev.owl

```diff
@@ -16452,6 +16452,90 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
+        <obo:IAO_0000233>https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/117</obo:IAO_0000233>
+        <rdfs:label>number of 2Rb Anopheles gambiae chromosomal inversion</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043215 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043215">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the trans-specific inversion 2Rc in Anopheles gambiae.</obo:IAO_0000115>
+        <obo:IAO_0000116>2Rc is located on chromosome arm 2R:26780000-31450000 (location in the AgamP3 assembly; GenBank: GCF_000005575.2) as described in: White, B.J., Cheng, C., Sangaré, D., Lobo, N.F., Collins, F.H. and Besansky, N.J., 2009. The population genomics of trans-specific inversion polymorphisms in Anopheles gambiae. Genetics, 183(1), pp.275-288. PMID:19581444</obo:IAO_0000116>
+        <obo:IAO_0000233>https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/117</obo:IAO_0000233>
+        <rdfs:label>number of 2Rc Anopheles gambiae chromosomal inversion</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043216 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043216">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the trans-specific inversion 2Ru in Anopheles gambiae.</obo:IAO_0000115>
+        <obo:IAO_0000116>2Ru is described in: White, B.J., Cheng, C., Sangaré, D., Lobo, N.F., Collins, F.H. and Besansky, N.J., 2009. The population genomics of trans-specific inversion polymorphisms in Anopheles gambiae. Genetics, 183(1), pp.275-288. PMID:19581444</obo:IAO_0000116>
+        <obo:IAO_0000233>https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/117</obo:IAO_0000233>
+        <rdfs:label>number of 2Ru Anopheles gambiae chromosomal inversion</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043217 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043217">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the chromosomal inversion 2Rd in Anopheles gambiae.</obo:IAO_0000115>
+        <obo:IAO_0000116>2Rj is described in: George, P., Sharakhova, M.V. and Sharakhov, I.V., 2010. High‐resolution cytogenetic map for the African malaria vector Anopheles gambiae. Insect molecular biology, 19(5), pp.675-682. PMID:20609021</obo:IAO_0000116>
```

Comment:
Just noticed that this should be "2Rd is described in"

The citation is correct.

(Oh, I see Chris already saw this.)

Reply:
Resolved in "Fixed texual definition & editor note" commit.